api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -170,8 +178,15 @@
     public ResponseEntity<Resource> downloadFile(
             @ApiParam(value = "A job identifier", required = true) @PathVariable @NotBlank String jobId,
             @ApiParam(value = "A file name", required = true) @PathVariable @NotBlank String filename) throws IOException {
-        HttpHeaders responseHeaders = new HttpHeaders();
-        responseHeaders.add("X-Progress", "0%");
-        return new ResponseEntity<>(null, responseHeaders, HttpStatus.ACCEPTED);
+        Path file = Paths.get(fileDownloadPath + filename);
```

Comment:
We can't really just take the `filename` provided by the user and concatenate, but the user could send us anything (security risk). We need to validate and make sure the `filename` in fact matches one the job's output precisely.